### PR TITLE
Added validation for CredentialMode

### PR DESCRIPTION
**Pull request by bkhadars**

### CODE REVIEW

**Comment by clnperez** [comment]
lgtm
requesting a review from @mkumatag

File: pkg/types/validation/installconfig_test.go [hunk 1]
```diff
@@ -1073,6 +1080,39 @@ func TestValidateInstallConfig(t *testing.T) {
...
+			name: "invalid powervs platform mint credential mod",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Platform = types.Platform{
+					PowerVS: validPowerVSPlatform(),
+				}
+				c.CredentialsMode = types.MintCredentialsMode
+				return c
+			}(),
+			expectedError: `^credentialsMode: Unsupported value: "Mint": supported values: "Manual"$`,
```
Comment (mkumatag) [inline pkg/types/validation/installconfig_test.go:1114]: do we have a testcase for the happy path by mentioning manual mode?

**Review by mkumatag — CHANGES_REQUESTED** [review]
overall lgtm except for one minor comment for the test case addition.

**Pull request merged**